src/messages/MClientCaps.h

```diff
@@ -117,7 +120,8 @@ class MClientCaps : public Message {
       time_warp_seq(0),
       osd_epoch_barrier(0),
       oldest_flush_tid(0),
-      caller_uid(0), caller_gid(0) {
+      caller_uid(0), caller_gid(0),
+      sync(false) {
```

Comment:
Jewel didn't have btime or change_attr. Those fields would both be new as of kraken.

So to be clear, that means that we do need to worry about backward compatability vs. development builds since jewel? If not, then we can still call it struct version 9 and it should still work. If we care about sane behavior vs. earlier development builds though, then we probably will need to do as you suggest.